pkg/config/config.go

```diff
@@ -0,0 +1,125 @@
+/*
+Copyright 2022 the kube-rbac-proxy maintainers. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+	"net"
+	"net/http"
+	"net/url"
+	"time"
+
+	serverconfig "k8s.io/apiserver/pkg/server"
+	"k8s.io/apiserver/pkg/server/dynamiccertificates"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/brancz/kube-rbac-proxy/pkg/authn"
+	"github.com/brancz/kube-rbac-proxy/pkg/authz"
+	"github.com/brancz/kube-rbac-proxy/pkg/proxy"
+)
+
+type KubeRBACProxyConfig struct {
+	SecureServing *serverconfig.SecureServingInfo
+
+	KubeRBACProxyInfo *KubeRBACProxyInfo
```

Comment:
What is the difference between `KubeRBACProxyInfo` and `KubeRBACProxyConfig` or more explicitly what is the meaning of `Info`? Is this k8s specific?

Reply:
Yup, looks like so 😆